### ISSUE TITLE
1.19 Updated Mod Crashing

### ISSUE THREAD

**Issue by SapphireFox420** [Alex-the-666/AlexsMobs#1159]
I followed the directions for the Warning under the update and had Citadel installed and together they keep crashing Minecraft. They give no crash code. When they crash they crash the launcher as well.

**Comment by DinoFighter** [comment]
Same for me except the last two sentences.

**Comment by Orangetones** [comment]
Having this issue as well.

LOG:
12:54:10.354
Found mod file citadel-1.12.5-1.19.jar of type MOD with provider {mods folder locator at C:\Users\Symon\curseforge\minecraft\Instances\AlexMobs\mods}
12:54:10.954

Exception in thread "main" java.io.UncheckedIOException: java.io.IOException: Invalid paths argument, contained no existing paths: [C:\Users\Symon\curseforge\minecraft\Install\libraries\net\minecraft\client\1.19-20220607.102129\client-1.19-20220607.102129-srg.jar, C:\Users\Symon\curseforge\minecraft\Install\libraries\net\minecraft\client\1.19-20220607.102129\client-1.19-20220607.102129-extra.jar, C:\Users\Symon\curseforge\minecraft\Install\libraries\net\minecraftforge\forge\1.19-41.0.62\forge-1.19-41.0.62-client.jar]
	at cpw.mods.securejarhandler@2.0.3/cpw.mods.jarhandling.impl.Jar.<init>(Jar.java:82)
	at cpw.mods.securejarhandler@2.0.3/cpw.mods.jarhandling.SecureJar.from(SecureJar.java:70)
	at MC-BOOTSTRAP/fmlloader@1.19-41.0.62/net.minecraftforge.fml.loading.moddiscovery.ModJarMetadata.buildFile(ModJarMetadata.java:41)
	at MC-BOOTSTRAP/fmlloader@1.19-41.0.62/net.minecraftforge.fml.loading.moddiscovery.MinecraftLocator.scanMods(MinecraftLocator.java:37)
	at MC-BOOTSTRAP/fmlloader@1.19-41.0.62/net.minecraftforge.fml.loading.moddiscovery.ModDiscoverer.discoverMods(ModDiscoverer.java:74)
	at MC-BOOTSTRAP/fmlloader@1.19-41.0.62/net.minecraftforge.fml.loading.FMLLoader.beginModScan(FMLLoader.java:166)
	at MC-BOOTSTRAP/fmlloader@1.19-41.0.62/net.minecraftforge.fml.loading.FMLServiceProvider.beginScanning(FMLServiceProvider.java:86)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.1/cpw.mods.modlauncher.TransformationServiceDecorator.runScan(TransformationServiceDecorator.java:112)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.1/cpw.mods.modlauncher.TransformationServicesHandler.lambda$runScanningTransformationServices$8(TransformationServicesHandler.java:100)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.HashMap$ValueSpliterator.forEachRemaining(HashMap.java:1779)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575)
	at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260)
	at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616)
	at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:622)
	at java.base/java.util.stream.ReferencePipeline.toList(ReferencePipeline.java:627)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.1/cpw.mods.modlauncher.TransformationServicesHandler.runScanningTransformationServices(TransformationServicesHandler.java:102)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.1/cpw.mods.modlauncher.TransformationServicesHandler.initializeTransformationServices(TransformationServicesHandler.java:55)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.1/cpw.mods.modlauncher.Launcher.run(Launcher.java:87)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.1/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.1/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.1/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@1.1.1/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
Caused by: java.io.IOException: Invalid paths argument, contained no existing paths: [C:\Users\Symon\curseforge\minecraft\Install\libraries\net\minecraft\client\1.19-20220607.102129\client-1.19-20220607.102129-srg.jar, C:\Users\Symon\curseforge\minecraft\Install\libraries\net\minecraft\client\1.19-20220607.102129\client-1.19-20220607.102129-extra.jar, C:\Users\Symon\curseforge\minecraft\Install\libraries\net\minecraftforge\forge\1.19-41.0.62\forge-1.19-41.0.62-client.jar]
	... 25 more
12:54:10.954
Process crashed with exit code 1

**Comment by SapphireFox420** [comment]
I ended up finding a solution. I had to remove all my mods to my desktop, delete Minecraft in my Roaming folder, and clear all my past data. Then I reinstalled Forge and added all my back mods, and it worked just fine.

**Comment by Orangetones** [comment]
Can you explain this process is a bit more detail?  I just copy the mods to my desktop? How do I clear all my past data?  I assume I can just search for the roaming folder and delete minecraft there?  Then reinstall forge and copy the mods back to the correct directory?  What directory is that?  Sorry for the questions, as I am relatively new to this.

**Comment by SapphireFox420** [comment]
It's cool. I'm also kind of new at it so I'm not the best at explaining I took a screenshot of my Minecraft folder I deleted. To find it you search your computer for _%appdata%_ then yes open Minecraft and copy your mods to your desktop as well as any game saves you want to keep in the _saves_ folder. then I deleted the Minecraft folder highlighted to clear all game data. Then restart Minecraft in the launcher to let it regenerate the game files. Once it does that close the game and reinstall whichever you had Fabric or Forge. (I reinstalled Forge 1.19) Then open your Minecraft folder and create a mods folder if it doesn't generate one and add your mods back into it and any save files back into the saves folder. It should work. If it doesn't may need to add mods one at a time to see if they work. (Make sure if you do that to have partner mods together like Alex's Mobs and Citadel) .

I hope this helps.
![image](https://user-images.githubusercontent.com/105528418/176980312-1bdcdc69-7a69-4734-aac0-650be2bcec12.png)
